lib/runtime/addon.js

```diff
@@ -157,8 +157,7 @@ export default class Addon {
     forIn(this.pkg.dependencies, (version, pkgName) => {
       let pkgMainPath = resolve.sync(pkgName, { basedir: this.dir });
       let pkgJSONPath = path.resolve(findup('package.json', { cwd: pkgMainPath }));
-      // TODO this shouldn't need to be aware of dist
-      let pkgDir = path.join(path.dirname(pkgJSONPath), 'dist');
+      let pkgDir = path.dirname(pkgJSONPath);
```

Comment:
This seems potentially problematic - the `createAddonFromDirectory` method below takes this `pkgDir` and attempts to add the contents as an addon. This seems like it will try to add the root folder, which will have the uncompiled source, rather than the `dist/` folder, which will be compiled.